app/models/user.py

```diff
@@ -0,0 +1,86 @@
+from sqlalchemy import String, Integer, Enum, ForeignKey, Date, DECIMAL, Boolean, Text
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from uuid import UUID
+from enum import Enum as PyEnum
+from .base import Base
+from .category import Goal, TodayNote, CategoryProgress
+from .chat import Chat, Storage
+from .evaluation import (
+    RecommendedChallenge,
+    ComprehensiveEvaluation,
+    MonthlyAchievement,
+)
+
+
+# 성별 Enum 클래스
+class GenderEnum(PyEnum):
+    MALE = "M"
+    FEMALE = "F"
+
+
+# User 테이블
+class User(Base):
+    __tablename__ = "users"
+
+    email: Mapped[str] = mapped_column(String(255), unique=True, nullable=False)
+    name: Mapped[str] = mapped_column(String(100), nullable=False)
+    password: Mapped[str] = mapped_column(String(255), nullable=False)
+    sex: Mapped[GenderEnum] = mapped_column(
+        Enum(GenderEnum, name="gender_enum"), nullable=False
+    )
+    age: Mapped[int] = mapped_column(Integer)
+    job: Mapped[str] = mapped_column(String(100), nullable=False)
```

Comment:
ERD에서는 null이 가능한 값입니당

```suggestion
    job: Mapped[str] = mapped_column(String(100))
```

Reply:
수정하였고 ERD상에서 NN되있는거 일괄적으로 nullable=False처리하였습니다(통일성)